Themes/default/css/index.css

```diff
@@ -3522,6 +3487,11 @@ h3.catbg .generic_icons::before, h3.catbg .icon {
 	vertical-align: middle;
 	margin: 0 5px 0 0;
 }
+.cat_bar + .windowbg2, .cat_bar + .windowbg, .cat_bar + .roundframe {
```

Comment:
I'm just curious related to this and missing .information + .windowbg/2 rest is nice ^^